pypesto/ensemble/ensemble.py

```diff
@@ -1157,3 +1184,33 @@ def get_percentile_label(percentile: Union[float, int, str]) -> str:
             percentile_str += '...'
         percentile = percentile_str
     return f'{PERCENTILE} {percentile}'
+
+
+def calculate_cutoff(result: Result, percentile: float = 0.95):
+    """
+    Calculate the cutoff of the ensemble.
+
+    Based on the number of parameters of the problem. Based on the
+    assumpiton that the difference of the NLLs of the true and optimal
+    parameter is chi^2 distributed with n_theta degress of freedom.
+
+    Parameters
+    ----------
+    result: The optimization result from which to create the ensemble.
+    percentile: The percentile of the chi^2 distribution. Between 0 and 1.
+    Higher values will result in a more lax cutoff. If the value is greater
+    than 1, the cutoff will be returned as np.inf.
```

Comment:
Different docstring format?

```suggestion
    result:
        The optimization result from which to create the ensemble.
    percentile:
        The percentile of the chi^2 distribution. Between 0 and 1.
        Higher values will result in a more lax cutoff. If the value is greater
        than 1, the cutoff will be returned as np.inf.
```